spec/helpers/util.js

```diff
@@ -60,6 +60,14 @@ function isIE() {
     return ((navigator.appName === 'Microsoft Internet Explorer') || ((navigator.appName === 'Netscape') && (new RegExp('Trident/.*rv:([0-9]{1,}[.0-9]{0,})').exec(navigator.userAgent) !== null)));
 }
 
+function isEdge() {
```

Comment:
For now, you only use it to retrieve the version number and never to actually test if the current browser is Edge. Am I wrong?